samples/testing/states/src/StatePrep.qs

```diff
@@ -0,0 +1,18 @@
+/// # Summary
+/// Prepares the state 1/2 (|00⟩ + i|01⟩ - |10⟩ - i|11⟩) = |-⟩ ⊗ |i⟩.
+operation PrepareStateWithComplexPhases(qs : Qubit[]) : Unit {
+    H(qs[0]);
+    Z(qs[0]);
+    H(qs[1]);
+    S(qs[1]);
+}
+
+/// # Summary
+/// Prepares the state 1/2 (-|00⟩ - i|01⟩ + |10⟩ + i|11⟩) = -|-⟩ ⊗ |i⟩.
```

Comment:
Maybe I'm just not used to the notation, but I would've expected the product version of the state to be either `-(|-⟩ ⊗ |i⟩)` or `-|-⟩ ⊗ -|i⟩` to drive home the fact that the -1 phase is global.

Reply:
In the second notation the minuses cancel each other out? 
It's just like regular multiplication, the minus can go in front of the whole thing or of any one of the terms, and here I'm preparing the state as (-|-⟩) ⊗ |i⟩ to get the global phase out of the first term, so it looked more natural. I can add the term `-(|-⟩ ⊗ |i⟩)` between the two terms I have right now to show the line of thought